ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalFoundationConsent.cshtml.cs

```diff
@@ -32,6 +32,12 @@ public async Task<IActionResult> OnPostAsync(int id)
 				_ => LegalRequirements.FoundationConsent
 			};
 
+			var (returnPage, fragment, back) = GetReturnPageAndFragment();
+			if (ReturnPage(returnPage))
```

Comment:
Could move this into the LegalModelBase.cs

Reply:
True!